docs/artefacts/onramps.md

```diff
@@ -404,6 +396,14 @@ Supported configuration options are:
 - `host` - The IP to listen on
 - `port` - The Port to listen on
 
+Set metadata variables:
+
+- `$binary` - `true` if the incoming websocket message came as binary (`false` otherwise)
```

Comment:
should we namepsace websocket meta-variables under `request`?  Since websockets are not really tied to the request/response semantics.

Or perhaps we can prefix it with the onramp name (`ws`) like how `$kafka_key` was added recently. Ideally, meta-variables everywhere would be namespaced more descriptively -- maybe something to tackle when we add a proper deployment language (eliminating the current onramp/offramp yaml).

Reply:
I don't think `request.*` is the right namespace to use here. `ws.message.format` would be a more appropriate namespace. But i think we should tackle this problem in a single sweep across all onramps/offramps setting/using meta variables, so we introduce some consistency that makes it more memorable, usable and expectable for users.

Thus skipping this as part of `0.9`.